lib/decidim/vocdoni/admin_engine.rb

```diff
@@ -13,6 +13,7 @@ class AdminEngine < ::Rails::Engine
         resources :wallets, only: [:new, :create]
         resources :elections do
           member do
+            get :publish_page
```

Comment:
I understand that we can't have it like 

```suggestion
            get :publish
```

as that would generate a conflict with the next line (`put :publish`), right? I say it because it's cleaner, to have it as /publish instead of /publish_page (at least in the URL).

Reply:
@andreslucena I didn't get this point very well. We have two actions in the controller:

`publish_page` is the get action showing the html
 `publish` is the put action to publish the election

We need two methods, just like new and create. I can't make the URL for publish_page the same as for publish. Maybe it would be preferable to change the URL? For example publish_election instead of publish page

Reply:
I think you understood me perfectly! The create/new example is spot on. 

>  Maybe it would be preferable to change the URL? For example publish_election instead of publish page

Yes, that would be great. `publish_page` I think it could introduce some confusion (there's a module `decidim-pages` that has not relation with this feature), and `publish_election` is more clear. Can you change that please?